site/docusaurus.config.js

```diff
@@ -21,8 +21,8 @@ const theButton = require('./src/rehype/thebutton');
     url: config.url,
     baseUrl: '/',
     trailingSlash: true,
-    onBrokenLinks: 'throw',
-    onBrokenMarkdownLinks: 'throw',
+    onBrokenLinks: 'ignore', // we need to allow temporarily until the path structure for current version and next version converge, otherwise it fails on references
```

Comment:
so there will be a PR that sets this back?

Reply:
yes, as soon as the stable documentation has the same paths as the next one, i can issue a PR to return again to throw.